third_party/istio-1.0.0/istio.yaml

```diff
@@ -1111,22 +341,13 @@ data:
             - NET_ADMIN
           privileged: true
         restartPolicy: Always
-
+      
       containers:
       - name: istio-proxy
-        # PATCH #2:  Add a prestop sleep.
```

Comment:
I think we can't keep that because we don't have control over how the user may install Istio.  I leave the bug open to track & fix that issue, which may or may not be there anymore (since it was an issue with Istio 0.6)

(see comment in https://github.com/knative/serving/pull/1771#issue-205289734)

Reply:
Good to know. Thanks!